modules/op-release-notes-1-17.adoc

```diff
@@ -260,3 +260,26 @@ c
 * Before this update, when you used the `tkn bundle` command to read the content of a large-sized Tekton bundle, the command might fail. With this update, the `tkn` command-line utilities correctly handles large Tekton bundles.
 
 * Before this update, when you used the `tkn` command-line utility to view a completed pipeline run, tasks that were skipped because of conditions you set up in the pipeline were displayed as `Succeeded(Completed)`. With this update, they are no longer displayed as completed.
+
+[id="release-notes-1-17-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.17.1
+
+With this update, {pipelines-title} General Availability (GA) 1.17.1 is available on {OCP} 4.15 and later versions.
+
+[id="fixed-issues-1-17-1_{context}"]
+=== Fixed issues
+
+* Before this update, if you defined a matrix task that included both regular parameters and `matrix` parameters, the `tekton-pipelines-controller` component crashed and logged a segmentation fault message. If the `Task` custom resource (CR) was not removed, the component continued to crash and did not run any pipelines. With this update, the controller no longer crashes in such cases.
+
+* Before this update, in some cases the {tekton-chains} controller repeatedly crashed, making the {tekton-chains} component unusable. With this update, the controller no longer crashes.
+
+* Before this update, the `buildah` task in the `openshift-pipelines` namespace did not allow spaces in the `BUILD_EXTRA_ARGS` parameter. With this update, the `buildah` task allows passing spaces in the `BUILD_EXTRA_ARGS` parameter.
++
+.Example of the BUILD_EXTRA_ARGS parameter with spaces in its value
+[source,yaml]
+----
+# ...
+   - name: BUILD_EXTRA_ARGS
+     value: '--build-arg FOO="abc def"'
```

Comment:
[IBM Style Guide](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#f): FOO. Do not use. This term is an acronym of a profanity that is sometimes used by developers in code. If you see this term in sample code, revise the example or do not publish it. Rewrite any text to omit the term.